homeassistant/components/climate/services.yaml

```diff
@@ -6,6 +6,8 @@ set_aux_heat:
   target:
     entity:
       domain: climate
+      supported_features:
+        - 64 # ClimateEntityFeature.AUX_HEAT
```

Comment:
Since it's all enum's, can't we just put AUX_HEAT here, and it is translated to 64 ?

Reply:
Just hassfest is enough, and I would just hardcode it.

Reply:
How is just hassfest enough, we need to send this to frontend too and we don't want frontend to have hardcoded copies of core enums?